### ISSUE TITLE
Question about command function in simple .jsx widgets

### ISSUE THREAD

**Issue by kknopper** [felixhageloh/uebersicht#370]
Hey @felixhageloh! 

Quick question about the command function in simple .jsx widgets.

Here you talked about being able to skip using the updateState function if your command function just returns a simple object.
https://github.com/felixhageloh/uebersicht/issues/355#issuecomment-466323699

But when I try this, I'm unable to get any output. I've tried dispatching my data, and just returning data, but neither seem to work. The value of output is always undefined.

Is updateState required now? Or am I just doing something wrong?

I'm currently on übersicht 1.4 running Mojave

```js
export const command = (dispatch) => {
	let output = 'Hello World';
	return output;
	//dispatch({type: "data", output}); This doesn't work either (instead of return)
}

export const refreshFrequency = 1000 // ms

export const render = ({ output, error }) => {
	return error ? (
		<Error>
			Something went wrong: <strong>{String(error)}</strong>
		</Error>
	) : (
	<main>
		{console.log('TEST', output)} //logs output as undefined
		<h1>{output}</h1>
	</main>
)}

export const className = `
	left: 20px;
	bottom: 150px;
	color: #fff;
	position: absolute;
`
```

**Comment by oscarmorrison** [comment]
Hey @kknopper I would like to hear from @felixhageloh as well on this. But I was unabled to get it working without the update state. To get it to work, i did something simple like so

<img width="449" alt="image" src="https://user-images.githubusercontent.com/1651212/60637066-43039980-9e5c-11e9-81a6-ad4b9b4e8c22.png">

**Comment by felixhageloh** [comment]
just had a look and this is bug! Will fix it for the next release. For now you can use the workaround of using dispatch

**Comment by oscarmorrison** [comment]
Thanks @felixhageloh

**Comment by felixhageloh** [comment]
sorry for closing this prematurely: an update that fixes this (v 1.4.60) has finally been released today!

**Comment by kknopper** [comment]
Thanks!
